hbase-server/src/main/java/org/apache/hadoop/hbase/master/region/MasterRegion.java

```diff
@@ -284,18 +284,21 @@ public static LocalRegion create(LocalRegionParams params) throws IOException {
     Configuration conf = new Configuration(baseConf);
     CommonFSUtils.setRootDir(conf, rootDir);
     CommonFSUtils.setWALRootDir(conf, walRootDir);
-    LocalRegionFlusherAndCompactor.setupConf(conf, params.flushSize(), params.flushPerChanges(),
+    MasterRegionFlusherAndCompactor.setupConf(conf, params.flushSize(), params.flushPerChanges(),
       params.flushIntervalMs());
     conf.setInt(AbstractFSWAL.MAX_LOGS, params.maxWals());
     if (params.useHsync() != null) {
       conf.setBoolean(HRegion.WAL_HSYNC_CONF_KEY, params.useHsync());
     }
+    if (params.useMetaCellComparator() != null) {
```

Comment:
We need to add this in here?

Reply:
This was missing?

Reply:
For used as procedure store, this is not necessary. This is for preparing for storing root table in the future, as for root table, the row key is the same with meta table, and we need to use a special comparator. I think it is fine to change the comparator in the future?

Reply:
ok